tests/ha/ctdb.pm

```diff
@@ -20,6 +20,9 @@ use version_utils 'is_sle';
 use utils qw(systemctl file_content_replace);
 
 sub run {
+    # Exit of this module if we are in a maintenance update not related to samba
+    return 1 if check_maintenance_update('samba');
```

Comment:
This line sounds weird for me, it sounds like if we exit the test if `samba` maintenance update is found.

Maybe easier to understand if you do a `return 0` instead of `return 1` L515 in `lib/hacluster.pm` and a `return 1 if !check_maintenance_update('samba');` here (and in the other files also), WDYT?

Reply:
In that case, better to change the name of the function to something like `is_not_maintenance_update('samba')` :P

Reply:
I agree that it's more readable like that, I'm triggering tests with it. Thx

Reply:
> In that case, better to change the name of the function to something like `is_not_maintenance_update('samba')` :P

If the function is not used to check a maintenance update later yes maybe better/easier to simply rename the function :D

Reply:
> In that case, better to change the name of the function to something like `is_not_maintenance_update('samba')` :P

Totally agree, much simpler eheh